content/akeneo-event-platform/concepts.md

```diff
@@ -0,0 +1,229 @@
+| `dataschema`         | Identifies the schema that `data` adheres to |
+| `time`         | Timestamp of when the event happened |
+| `data`         | The event JSON payload |
+
+For more information, consult the [CloudEvents spec attributes](https://github.com/cloudevents/spec/blob/v1.0.2/cloudevents/spec.md).
+
+## Event Overview
+
+Here is the list of all the available events :
+
+|  Event family       |  event |
+|----------|-----------------------------------------------|
+| **product** | `com.akeneo.pim.v1.product.created`<br/>`com.akeneo.pim.v1.product.updated`<br/>`com.akeneo.pim.v1.product.deleted`     |
+| **product model** | `com.akeneo.pim.v1.product-model.created`<br/>`com.akeneo.pim.v1.product-model.updated`<br/>`com.akeneo.pim.v1.product-model.deleted`      |
+| **asset** | `com.akeneo.pim.v1.asset.created`<br/>`com.akeneo.pim.v1.asset.updated`<br/>`com.akeneo.pim.v1.asset.deleted`     |
+| **reference entity** | `com.akeneo.pim.v1.reference-entity-record.updated`<br/>`com.akeneo.pim.v1.reference-entity-record.deleted`   |
+| **category** | `com.akeneo.pim.v1.category.created`<br/>`com.akeneo.pim.v1.category.updated`<br/>`com.akeneo.pim.v1.category.deleted`     |
+| **family** | `com.akeneo.pim.v1.family.updated`<br/>`com.akeneo.pim.v1.family.deleted`     |
+| **attribute** | `com.akeneo.pim.v1.attribute.updated`<br/>`com.akeneo.pim.v1.attribute.deleted`<br/>`com.akeneo.pim.v1.attribute-option.updated`<br/>`com.akeneo.pim.v1.attribute-option.deleted`<br/>`com.akeneo.pim.v1.attribute-group.updated`<br/>`com.akeneo.pim.v1.attribute-group.deleted`     |
+| **connection** | `com.akeneo.pim.v1.app.deleted`<br/>`com.akeneo.pim.v1.connection.deleted`     |
+
+Events are dispatched regardless of their triggers. Possible triggers include:
+
+- **Interface Change**: When a client modifies data directly through the user interface.
+- **API Change**: When data is updated via the API, whether by the client or through an app/connection.
+- **Mass Action Change**: When multiple entities are updated at once, typically during bulk operations.
+- **Rules Change**: When the rule engine is used to make modifications.
+- **Import Change**: When data is brought in via the import engine.
+
+For more event specific details, consult our [event list page](/akeneo-event-platform/available-events.html).
+
+## Retry policies
+
+In distributed systems, ensuring reliable message processing is crucial. Retry policies help manage transient failures and guarantee message delivery even when encountering issues.
+
+Here, we outline the key **retry policies** implemented in our system.
+
+### ACK timeout
+
+ACK timeout ensures that messages are acknowledged within a specified period. In our system, the ACK timeout is set to **`3 seconds`**. If an acknowledgment isn't received within this timeframe, the message is retried. This mechanism prevents message loss due to slow consumers or network delays, ensuring timely processing.
```

Comment:
Being inspired by shopify documentation we might not document "ack" terminology, as it unnecessarily leak our underlying components (pubsub).

https://shopify.dev/docs/apps/build/webhooks/subscribe/https#respond-with-a-200-ok-quickly

Plus, we can document "1 second" timeout too, even if internally we put it at 3 sec it forces https destination to be quick. Wdyt ?

Reply:
We already receive demands to increase the timeout from a client (middleby i think)
It's maybe better to be clear 1 or 3s to avoid the fact customer say it's documented 1s but 3s work fine